sdk/identity/identity-cache-persistence/package.json

```diff
@@ -58,14 +58,11 @@
   "bugs": {
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
-  "engine": {
```

Comment:
Is this removal intentioanl? @sadasant

Reply:
yes, it seems unnecessary. Let me ask @willmtemple , does it make sense to have both `engine` and `engines`? I noticed this in other packages too

Reply:
It should just be `"engines"`. I probably copied this from somewhere in our repo that had both.